cargo-dist/templates/installer/installer.sh.j2

```diff
@@ -28,11 +28,9 @@ EORECEIPT
 # Are we happy with this same path on Linux and Mac?
 RECEIPT_HOME="${HOME}/.config/{{ app_name }}"
 
-# glibc provided by our Ubuntu 20.04 runners;
-# in the future, we should actually record which glibc was on the runner,
-# and inject that into the script.
-BUILDER_GLIBC_MAJOR="2"
-BUILDER_GLIBC_SERIES="31"
+# glibc provided by the system(s) which built these binaries
+BUILDER_GLIBC_MAJOR="{{ runtime_conditions.linux.glibc.major }}"
+BUILDER_GLIBC_SERIES="{{ runtime_conditions.linux.glibc.series }}"
```

Comment:
These are currently nullable, right? And Extremely bad/chaotic things would happen if they were null? (in all the kinds of installers)

If we don't want to handle that in this PR, we probably want somewhere in the platform code default this to 2.31 when no info is available

Reply:
Yeah, that's one of the things I was thinking about with reusing these types. Specifically, we always calculate a value here, but the type itself is an option - it's probably dangerous to rely on it always being `Some()` even if we always assign to it.